api/src/pcapi/core/users/models.py

```diff
@@ -278,6 +282,10 @@ def add_underage_beneficiary_role(self) -> None:
     def add_test_role(self) -> None:
         self._add_role(UserRole.TEST)
 
+    def add_anonimized_role(self) -> None:
```

Comment:
```suggestion
    def add_anonymized_role(self) -> None:
```
Ici c'est le i/y et pas le s/z !